deployment/deploy-az-demo-bootstrapper.ps1

```diff
@@ -0,0 +1,37 @@
+# ------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All rights reserved.
+#  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+# ------------------------------------------------------------
+Param(
+    [string]
+    [Parameter(mandatory=$true)]
+    $ApplicationName
+)
+
+# ----- Copy scripts from source location
+$baseLocation = "https://raw.githubusercontent.com/suneetnangia/distributed-az-edge-framework/feature/docs-update"
```

Comment:
Is this really the right url?

Reply:
No, once I finish the testing (need to verify OPCUA telemetry), last update to this branch will be to change the url.

Reply:
Changing PR to draft to signify WIP.